pandas/core/generic.py

```diff
@@ -5799,6 +5800,123 @@ def infer_objects(self: FrameOrSeries) -> FrameOrSeries:
             )
         ).__finalize__(self)
 
+    def convert_dtypes(
+        self: FrameOrSeries,
+        infer_objects: bool_t = True,
+        convert_string: bool_t = True,
+        convert_integer: bool_t = True,
+        convert_boolean: bool_t = True,
+    ) -> FrameOrSeries:
+        """
+        Convert columns to best possible dtypes, optionally using dtypes supporting
+        ``pd.NA``.
+
+        For object-dtyped columns, if ``infer_objects`` is ``True``, use the inference
```

Comment:
I would move these to the Notes section I think

Reply:
done